app/views/content/train-to-be-a-teacher/if-your-application-is-unsuccessful.md

```diff
@@ -69,7 +69,7 @@ Find out more about [how visa sponsorships work](/non-uk-teachers/train-to-teach
 
 ## Get advice on what to do next
 
-A teacher training adviser can help you improve your application if you have been unsuccessful.
+A teacher training adviser can help you improve your application if you have been unsuccessful for free.
```

Comment:
I don't think this makes sense - `unsuccessful for free`? I think the sentence needs rewriting?